src/app/ubs/ubs/components/ubs-order-details/ubs-order-details.component.html

```
@@ -11,8 +11,10 @@ <h5 *ngIf="!currentLocation" class="error-text">{{ 'order-details.title-without-
           </button>
         </div>
       </div>
-      <div class="validMes" *ngIf="orderDetailsForm?.controls.bags.errors">
-        <small class="text-danger" *ngFor="let error of bagErrors">{{ error.message | translate: { value: error.value } }} <br /> </small>
+      <div class="validMes">
+        <small class="text-danger" *ngIf="orderDetailsForm?.controls.bags.errors as errors">{{
```

HTMLHint (1.5.0)

[error] The attribute name of [ *ngIf ] must be in lowercase.

(attr-lowercase)